### DISCUSSION TITLE
Multiple Inputs for touchscreen users.

### DISCUSSION THREAD

**Discussion by timelessnesses** [ppy/osu#14823] [Ideas] [answered]
so yeah.<br>
This feature is required because it is really hard to play some map that have circle stack on them and you can't click all of them in time or there are stream that you need to click and you don't have much time to touch the keyboard. So I suggest there should be multiple input where you can use your other hand to make cursor give a click.<br>
If you don't understand try to see osu!droid plays videos and think how hard it is if there's no multiple inputs.<br>
:D

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/4201
